src/main/java/de/zalando/aruha/nakadi/service/StrategiesRegistry.java

```diff
@@ -9,20 +9,13 @@
 public class StrategiesRegistry {
 
     public static final PartitionStrategyDescriptor HASH_PARTITION_STRATEGY =
-            new PartitionStrategyDescriptor("hash", "This strategy will use the event " +
```

Comment:
I think so. This is duplicating the description from the API. It increases the cost of maintenance and provides nothing in return, since this is fixed. I think it would be better to keep documentation in a single place, the API yml.